### DISCUSSION TITLE
Listener Stats Expanded

### DISCUSSION THREAD

**Discussion by Cpnkrk** [advplyr/audiobookshelf#2149] [Ideas]
Is there a way to add how many times a book has been completed? Like a tally mark or something, this would be helpful for repeatedly listened to audiobooks. 

For example: Book: Lord of the Rings: 2x`s it could be something stated in the information section similar to how there is the current progress bubble there could be a number of times completed bubble.

**Comment by nichwall** [comment]
There's an open feature request for it if you want to put a thumbs up on it (advplyr sorts by thumbs up when looking for ideas of what to implement next)

https://github.com/advplyr/audiobookshelf/issues/1653
